app/src/main/java/de/westnordost/streetcomplete/osm/cycleway/CyclewayParser.kt

```diff
@@ -94,6 +94,7 @@ private fun parseCyclewayForSide(
     val cycleway = tags[cyclewayKey]
     val cyclewayLane = tags["$cyclewayKey:lane"]
     val isSegregated = tags["$cyclewayKey:segregated"] != "no"
+    val isCyclingOkOnSidewalk = tags["sidewalk$sideVal:bicycle"] == "yes" || tags["sidewalk:both:bicycle"] == "yes"
```

Comment:
> tags["sidewalk:both:bicycle"] == "yes"

Don't check for `both` specifically, the sides are expanded in a parsing step before, in `expandRelevantSidesTags`. Add `result.expandSidesTags("sidewalk", "bicycle", true)` there instead.